prow/config.yaml

```diff
@@ -1937,6 +1937,38 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  - name: pull-kubernetes-e2e-gce-dind-ipv6
```

Comment:
Sorry couple dumb questions: how robust will this job be? does it block PR? why is post-commit jobs not enough?

Reply:
Good question. I'd have to look for guidance from the community on whether this should eventually (after some soak time) be gating/blocking, and whether a post-commit job is good enough. Our goal is for the tests to bring IPv6 to Beta status (I don't know if there is a standard for that), and to detect possible future regression bugs with a "reasonable" amount of time (maybe a 6-hour periodic is good enough for that?) In my estimation, a non-blocking pre-commit and every-6-hour post-commit would be good.

Reply:
> Our goal is for the tests to bring IPv6 to Beta status

Not sure the criteria either, maybe having all IPv6 CIs green on the release branch is sufficient?

Don't have strong opinion here, my thought on non-blocking pre-commit is that people might not care whether it fails or not if it doesn't block their PRs being merged. Likely the post-commit will still be the only place we get the failing signal :)